Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -35,4 +48,36 @@ void BatteryStateOfChargeService::addData(const BatteryData& batteryData)
     // This is where you can update your variables
     // Hint: There are many different ways that the totalAmpHoursUsed can be updated
     // i.e: Taking a running average of your data values, using most recent data points, etc.
+
+    // The section below is updates the total Amphours used
+    if (dataPoints_ > 0)
```

Comment:
Since we're only checking if `dataPoints_` is 0 or not, I would suggest to use a boolean instead to indicate if it's the first run.